Cards/MadGraph5_aMCatNLO/Z/ZGto2NuG-1Jets_Bin-PTG-100_amcatnloFXFX-pythia8/ZGto2NuG-1Jets_Bin-PTG-100_amcatnloFXFX-pythia8.json

```diff
@@ -0,0 +1,26 @@
+{
+    "template": "NLO_run_card.dat",
+    "template_user": [],
+    "template_vars":{
+        "parton_shower": "pythia8",
+        "ickkw": "3",
+        "maxjetflavor": "5"
+    },
+    "model_params": "massZ.dat",
+    "model_params_user": [
+        "set run_card ptj 10",
+        "set run_card ptgmin 20"
```

Comment:
didn't we put these values 70 GeV below the LHE-level boundary? Shouldn't that be 30 instead of 20? Or will we increase all values to 80 because 70 GeV was still too close to the boundary.